packages/ui/src/components/organisms/Filter/styles.scss

```diff
@@ -0,0 +1,119 @@
+[data-fs-filter] {
+  // --------------------------------------------------------
+  // Design Tokens for Filter
+  // --------------------------------------------------------
+
+  // Title
+  --fs-filter-title-height                                : var(--product-listing-row-height);
```

Comment:
maybe we will need a global token to define `--product-listing-row-height` in the future.

Reply:
that's true. for now I'll replace for `var(--fs-spacing-6)`, since it's the value that the product-gallery style has for the token `--product-listing-row-height`.